regulations/settings/base.py

```diff
@@ -229,4 +229,7 @@
     'regulations.generator.sidebar.help.Help',
 )
 
+BUCKET = os.getenv('BUCKET')
+ACCESS_KEY_ID = os.getenv('ACCESS_KEY_ID')
+SECRET_ACCESS_KEY = os.getenv('SECRET_ACCESS_KEY')
```

Comment:
Should have pointed this out before, but it'd be helpful to namespace these a little better. You could imagine there's an S3 bucket dedicated to static assets, for example.